compiler/src/typed/typemod.re

```diff
@@ -104,7 +105,15 @@ let include_module = (env, sod) => {
     );
   let newenv = Env.include_module(mod_name, sod, env);
 
-  let od = {tinc_path: path, tinc_loc: sod.pinc_loc};
+  let mod_file =
```

Comment:
Just one question before I press approve.    When would the module file end in .wasm ?